akka-distributed-data/src/main/scala/akka/cluster/ddata/Replicator.scala

```diff
@@ -1473,6 +1471,8 @@ private[akka] abstract class ReadWriteAggregator extends Actor {
 
   def timeout: FiniteDuration
   def nodes: Set[Address]
+  def unreachable: Set[Address]
+  def reachableNodes: Set[Address] = nodes -- unreachable
```

Comment:
I'd really like to understand this. Do you have some reference? If I look at the implementation of `diff` it is (in trait `SetLike`):
```
def diff(that: GenSet[A]): This = this -- that
```
and `--` is implemented in the trait `Subtractable`. If I use `--` directly it will also use the implementation in `Subtractable`.